nixos/modules/profiles/installation-device.nix

```diff
@@ -10,10 +10,6 @@ with lib;
       ../installer/scan/detected.nix
       ../installer/scan/not-detected.nix
 
-      # Allow "nixos-rebuild" to work properly by providing
-      # /etc/nixos/configuration.nix.
-      ./clone-config.nix
-
```

Comment:
`nixos/modules/installer/virtualbox-demo.nix` was using `installer.cloneConfigExtra` from this import.

```
~/tmp/nixpkgs/nixpkgs-PR63773 2 $ nix-instantiate --arg nixpkgs '{ outPath=./.; revCount=999999; shortRev="ofborg"; }' .
/nixos/release-combined.nix -A tested
warning: dumping very large path (> 256 MiB); this may run out of memory
error: The option `installer' defined in `/Users/samuel/tmp/nixpkgs/nixpkgs-PR63773/nixos/modules/installer/virtualbox-demo.nix'
does not exist.
(use '--show-trace' to show detailed location information)
```

In addition, the cloneConfig system for installer isos is desired. Let's say I need to rebuild with a specific setting for doing something, there is currently a loss of function.

I think, instead, `installer.cloneConfig` should be set to `false` on image types where the installation media is also the installation target.

Reply:
I pushed a new version, which sets `installer.cloneConfig = false;` instead.

This only seemed to be necessary for `sd-image.nix`.